configuration/src/main/java/io/camunda/configuration/beans/RestoreProperties.java

```diff
@@ -5,13 +5,8 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.zeebe.restore;
+package io.camunda.configuration.beans;
 
 import java.util.List;
-import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.bind.DefaultValue;
 
-@ConfigurationProperties(prefix = "zeebe.restore")
-public record RestoreConfiguration(
-    @DefaultValue("true") boolean validateConfig,
-    @DefaultValue({"lost+found"}) List<String> ignoreFilesInTarget) {}
+public record RestoreProperties(boolean validateConfig, List<String> ignoreFilesInTarget) {}
```

Comment:
Why renaming RestoreConfiguration into RestoreProperties? Can we not keep the same name?